src/jit/codegenlegacy.cpp

```diff
@@ -20433,7 +20433,11 @@ regMaskTP           CodeGen::genCodeForCall(GenTreePtr  call,
     if (call->gtType == TYP_FLOAT || call->gtType == TYP_DOUBLE)
     {
 #ifdef _TARGET_ARM_
-        if (call->gtCall.IsVarargs())
+        bool UseIntReg = call->gtCall.IsVarargs();
```

Comment:
Assigned value is never used if ARM_SOFTFP. I think it is possible to write as follows:
#ifndef ARM_SOFTFP
   if (call->gtCall.IsVarargs())
#endif
  {
    ...